.github/workflows/quality-report.yml

```diff
@@ -36,3 +38,10 @@ jobs:
         env:
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          
+      - name: Increment Version
+        id: versioning
+        uses: Oliver-Binns/Versioning@1.0.0
```

Comment:
same as the above, should this not be `Oliver-Binns/Versioning@v1`

Reply:
@sandy-may left a comment stating the `v` in the annotation should be removed to reference the release tag correctly.